src/common/components/paymentMethods/creditCardForm/creditCardForm.tpl.html

```diff
@@ -95,11 +95,8 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Credit Card Paym
         <div class="panel-body">
           <h4 class="panel-title  border-bottom-small  visible">Billing Address</h4>
           <display-address address="$ctrl.donorDetails.mailingAddress"></display-address>
-          <div class="loading-wrap" ng-if="!$ctrl.donorDetails"><!-- TODO: this seems to be misaligned -->
-            <span class="loading loading1"></span>
-            <span class="loading loading2"></span>
-            <span class="loading loading3"></span>
-            <span class="loading loading4"></span>
+          <div style="position: relative">
```

Comment:
@riceguitar I added a few of these to get the loading indicators to be aligned right. You have any suggestions for making it cleaner?